pkgs/development/compilers/cmdstan/default.nix

```
@@ -38,6 +31,11 @@ stdenv.mkDerivation rec {
       src/test/interface/stansummary_test.cpp
   '';
 
+  preConfigure = ''
+    mkdir -p bin
+    ln -s ${stanc}/bin/stanc bin/stanc
```

Is this really required? Can't we remove the static path instead?

---

The makefile [requires that](https://github.com/stan-dev/cmdstan/blob/develop/make/stanc#L47) an executable to be at `bin/stanc`, and patching `bin/stanc$(EXE)` around gave me errors on tests. I guess it's easier to leave it as is.

---

```suggestion
    ln -s ${buildPackages.stanc}/bin/stanc bin/stanc
```
then I think we want to do this to make it cross friendly

---

Does. Sorry to have to rebase it for storage constraints.